sdk/eventgrid/Microsoft.Azure.Messaging.EventGrid.CloudNativeCloudEvents/README.md

```diff
@@ -64,6 +65,23 @@ var cloudEvent =
 await client.SendCloudNativeCloudEventAsync(cloudEvent);
 ```
 
+### Publish to an Event Grid Domain
+When publishing to an Event Grid Domain with CloudEvents, the CloudEvent `source` is used as the domain topic. The Event Grid service doesn't support using an absolute URI for a domain topic, so you would need to do something like the following to integrate with the CloudNative CloudEvents:
+```C# Snippet:CloudNativePublishToDomain
+CloudEvent cloudEvent =
+    new CloudEvent
+    {
+        Type = "record",
+        // Event Grid does not allow absolute URIs as the domain topic
+        Source = new Uri("test", UriKind.Relative),
+        Id = Recording.Random.NewGuid().ToString(),
```

Comment:
I'm guessing that `Recording` is related to our test framework.   I'm not sure this would be obvious to readers and may be taken as having more significance.   Should we consider assigning these to intermediate variables with dummy values for snippets and substituting the recording framework values for test execution?